src/Client.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
-namespace GoCardless\Enterprise;
-
-use GoCardless\Enterprise\Exceptions\ApiException;
-use GoCardless\Enterprise\Model\Creditor;
-use GoCardless\Enterprise\Model\CreditorBankAccount;
-use GoCardless\Enterprise\Model\Customer;
-use GoCardless\Enterprise\Model\CustomerBankAccount;
-use GoCardless\Enterprise\Model\Mandate;
-use GoCardless\Enterprise\Model\Model;
-use GoCardless\Enterprise\Model\Payment;
+namespace Lendable\GoCardlessEnterprise;
+
+use Lendable\GoCardlessEnterprise\Exceptions\ApiException;
```

Comment:
@alexlongshaw Your assumption is correct, this will be on a new major version. Very similar change is prepared for GoCardlessEnterpriseBundle too. The idea is to bring these two repositories inline with each other in terms of namespace.